### PR TITLE
issue-963 Add uploadedTime property to VideoModel

### PR DESCRIPTION
**Pull request by incjo**
Adding `uploadedTime` property to the `VideoModel` to bring it inline with the `PhotoModel` in order to make it easier to set the creation time for transferred videos.

### CODE REVIEW

**Review by jzacsh — APPROVED** [review]

File: extensions/data-transfer/portability-data-transfer-backblaze/src/test/java/org/datatransferproject/datatransfer/backblaze/videos/BackblazeVideosImporterTest.java [hunk 1]
```diff
@@ -111,7 +111,7 @@ public void testImportVideo() throws Exception {
         UUID jobId = UUID.randomUUID();
 
         VideoModel videoObject =
-                new VideoModel(title, videoUrl, description, encodingFormat, dataId, albumId, false);
+                new VideoModel(title, videoUrl, description, encodingFormat, dataId, albumId, false, null);
```
Comment (jzacsh) [inline extensions/data-transfer/portability-data-transfer-backblaze/src/test/java/org/datatransferproject/datatransfer/backblaze/videos/BackblazeVideosImporterTest.java:114]: could you make this change for anywhere that you're touching the params on a given model? (so I'm commenting on this line of this file as an example - but if you could make it on each line you're touching _anyway_ in-passing in this PR, that'd be appreciated!)

```suggestion
                new VideoModel(title, videoUrl, description, encodingFormat, dataId, albumId, false /*inTempStore*/, null /*uploadTime*/);
```

File: extensions/data-transfer/portability-data-transfer-koofr/src/main/java/org/datatransferproject/transfer/koofr/videos/KoofrVideosImporter.java [hunk 2]
```diff
@@ -35,7 +35,9 @@
 import org.datatransferproject.types.common.models.videos.VideosContainerResource;
 import org.datatransferproject.types.transfer.auth.TokensAndUrlAuthData;
 
-/** Imports albums and videos to Koofr. */
+/**
+ * Imports albums and videos to Koofr.
```
Comment (jzacsh) [inline extensions/data-transfer/portability-data-transfer-koofr/src/main/java/org/datatransferproject/transfer/koofr/videos/KoofrVideosImporter.java:39]: entirely-optional nit: just clarifying that it's not just generalized albums (even though it _could_ be, given the media vertical's complications)

```suggestion
 * Imports videos and their albums to Koofr.
```

**Review by jzacsh — APPROVED** [review]
LGTM - just left a couple comments for small changes.

File: extensions/data-transfer/portability-data-transfer-backblaze/src/test/java/org/datatransferproject/datatransfer/backblaze/videos/BackblazeVideosImporterTest.java (hunk 1 above)
Comment (incjo) [reply extensions/data-transfer/portability-data-transfer-backblaze/src/test/java/org/datatransferproject/datatransfer/backblaze/videos/BackblazeVideosImporterTest.java:114]: This seems like an anti-pattern to me and I'd rather not do that. IntelliJ already does this by default without cluttering up the code, e.g. 
image

Comment (jzacsh) [reply extensions/data-transfer/portability-data-transfer-backblaze/src/test/java/org/datatransferproject/datatransfer/backblaze/videos/BackblazeVideosImporterTest.java:114]: Got it - I'll add this to the list of things to get our codebase consistent/automated on (there's pros/cons to doing it, but I must've been mistaken that folks were agreed on doing it - so perfect thing to punt for some hackathon to find agreement on).

File: portability-types-common/src/main/java/org/datatransferproject/types/common/models/videos/VideoModel.java [hunk 3]
```diff
@@ -90,36 +93,46 @@ public String getName() {
     return super.getName();
   }
 
+  public Date getUploadedTime() {
```
Comment (wmorland) [inline portability-types-common/src/main/java/org/datatransferproject/types/common/models/videos/VideoModel.java:96]: Do we use any @Nullable annotations elsewhere? Can help IDEs pick up NPEs and similar.

**Review by wmorland — APPROVED** [review]
Happy for us to merge this

**Pull request merged**